src/teaching-element/PageBreak.vue

```diff
@@ -10,7 +10,7 @@
     </div>
     <span>
       <slot name="pageBreakIcon">
-        <span class="mdi mdi-chevron-down"></span>
+        <span class="break-icon mdi mdi-chevron-down"></span>
```

Comment:
I think the `.break-icon` class should be on a wrapper tag that contains the slot instead of inside the slot. Since its purpose is to hide the icon, the CSS rules should apply to any possible variation of the icon.